seacrowd/sea_datasets/snli/snli.py

```diff
@@ -0,0 +1,163 @@
+"""
+SNLI Indo is derived from the SNLI corpus, where the premise and hypothesis sentences are translated directly from English to Indonesian using the Google Cloud Translation API. The SNLI corpus is divided into three sets, namely train, development, and test set. The translation process is applied to all the premise and hypothesis sentences in all the three sets. This ensures that the number of sentence pairs obtained is the same as the original SNLI dataset, namely 570k sentence pairs. A filtering process is carried out to remove incomplete sentence pairs and those with a gold label `-`. As a result, 569,027 sentence pairs are obtained.
+"""
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import jsonlines
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{suwija2023snli,
+  author       = "Suwija Putra, I Made 
+        and Siahaan, Daniel 
+        and Saikhu, Ahmad",
+  title        = "SNLI Indo: A recognizing textual entailment dataset in Indonesian derived from the Stanford Natural Language Inference dataset"
+  year         = "2024",
+  journal      = "Data in Brief",
+  volume       = "52",
+  pages        = "109998",
+  publisher    = "Elsevier",
+  doi          = "https://doi.org/10.1016/j.dib.2023.109998",
+  url          = "https://www.sciencedirect.com/science/article/pii/S2352340923010284",
+}
+"""
+
+_DATASETNAME = "snli"
+
+_DESCRIPTION = """\
+The SNLI Indo dataset is derived from the SNLI corpus by translating each premise and hypothesis sentence from English to Indonesia via the Google Cloud Translation API. Premise sentences are crawled image captions from Flickr, and hypothesis sentences are manually created through crowdsourcing. Five annotators are assigned per sentence pair to label the inference relationship as entailment (true), contradiction (false) or neutral (undetermined).
+"""
+
+_HOMEPAGE = "https://data.mendeley.com/datasets/k4tjhzs2gd/1"
+
+_LANGUAGES = ["ind"]
+
+_LICENSE = Licenses.CC_BY_4_0.value
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: {
+        "train": "https://data.mendeley.com/public-files/datasets/k4tjhzs2gd/files/ee45b2bb-e2ea-47b7-bec4-b6653c467d27/file_downloaded",
+        "val": "https://data.mendeley.com/public-files/datasets/k4tjhzs2gd/files/23aff85c-ff72-48b6-aba1-c1dd5dac216b/file_downloaded",
```

Comment:
The validation data URL is identical to the test data URL. Is this intentional and as per how the dataset should be used?

Reply:
Thanks for the notice. Revised.